VSRAD.Package/Utils/VsEditor.cs

```diff
@@ -53,21 +53,17 @@ public static void OpenFileInEditor(SVsServiceProvider serviceProvider, string p
             {
                 var dte = serviceProvider.GetService(typeof(DTE)) as DTE;
                 Assumes.Present(dte);
-                var activeFile = dte.ActiveDocument;
-                dte.ItemOperations.OpenFile(path); // open requested file in VS editor
+                dte.ItemOperations.OpenFile(path);
 
-                if (!string.IsNullOrEmpty(lineMarker))
-                {
-                    var lineNumber = GetMarkedLineNumber(path, lineMarker);
+                if (string.IsNullOrEmpty(lineMarker)) return;
 
-                    var textManager = serviceProvider.GetService(typeof(SVsTextManager)) as IVsTextManager2;
-                    Assumes.Present(textManager);
+                var lineNumber = GetMarkedLineNumber(path, lineMarker);
 
-                    textManager.GetActiveView2(1, null, (uint)_VIEWFRAMETYPE.vftCodeWindow, out var activeView);
-                    activeView.SetCaretPos(lineNumber, 0);
-                }
+                var textManager = serviceProvider.GetService(typeof(SVsTextManager)) as IVsTextManager2;
+                Assumes.Present(textManager);
 
-                dte.ItemOperations.OpenFile(activeFile.FullName); // preserving old active document
```

Comment:
Just comment this line. No other changes needed.

Reply:
Thanks, implemented in 4c454f4